frontend/src/components/landingPage/MoreInformation.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div id="moreInfoPage">
+    <app-header></app-header>
+    <h5></h5>
+    <div id="moreInfoDiv">
+      <div id="moreInfoTextDiv">
+        <p v-if="this.receivedId != this.errorStr">ID : {{ receivedId }}</p>
```

Comment:
Remove ID

Reply:
This has been fixed and commit has been pushed.